### ISSUE TITLE
Allowing for single responses

### ISSUE THREAD

**Issue by uwidev** [nathanhoad/godot_dialogue_manager#22]
I kept on running into a weird problem during debugging where the responses were not appearing. The response I wrote just straight up was not appearing. 

![image](https://user-images.githubusercontent.com/10101319/154733592-a0aca252-4dd4-4aa6-99c7-66c863f3dce8.png)

Turns out that with only one response, it just straight up isn't rendered. You need minimum 2 responses for it to work. I'm not too sure if this is an issue with the example dialogue bubble or an issue with the dialogue manager itself, but it's something that needs looking into.

Why would you ever want a single response? To signal to the player that they have no choice, that something is inevitable, and that it is them is doing it. It's more active and engaging than passive dialogue text.

**Comment by wayrboy** [comment]
do function()

**Comment by nathanhoad** [comment]
Yeah that's a good point. I'll remove the stuff that bypasses single responses.
